library/sinks/OpenAI.ts

```diff
@@ -113,6 +116,16 @@ export class OpenAI implements Wrapper {
     return "openai";
   }
 
+  private getRoute() {
```

Comment:
maybe extract as a helper function? we will probably expand AI usage coverage right